<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGui">
      <attribute name="title">
       <string>GUI settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayoutTabGui">
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBoxAppFontSize">
         <property name="toolTip">
          <string>Application font size</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <spacer name="verticalSpacerGUI">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelLanguage">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxLanguage">
         <property name="toolTip">
          <string>Select the GUI language (requires a restart to take effect!)</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>German (de)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English (us)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxStyle">
         <property name="toolTip">
          <string>Select the GUI style</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAppFont">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="fontComboBoxAppFont">
         <property name="toolTip">
          <string>Application font</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelAppFontSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayoutGuiFlags">
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBoxNativeFileDialogs">
           <property name="toolTip">
            <string>Use the platform's native file dialogs or the Qt built-in ones</string>
           </property>
           <property name="text">
            <string>Use native file dialogs</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxMaximizeWindows">
           <property name="toolTip">
            <string>Maximize connection windows</string>
           </property>
           <property name="text">
            <string>Maximize windows</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBoxQuiet">
         <property name="toolTip">
          <string>Suppress output logged to 'stdout'</string>
         </property>
         <property name="text">
          <string>Suppress log output</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVnc">
      <attribute name="title">
       <string>VNC defaults</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxBilinearFiltering">
         <property name="toolTip">
          <string>Use bilinear filtering on scaled displays?</string>
         </property>
         <property name="text">
          <string>Bilinear filtering</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" rowspan="2" colspan="5">
        <spacer name="verticalSpacerVnv">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxScaledDisplay">
         <property name="toolTip">
          <string>Scale the display to the connection window size (recommended)?</string>
         </property>
         <property name="text">
          <string>Scaled display</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QLabel" name="labelChdmanWarningPixmap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap>:/images/warning.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxKeepAspect">
         <property name="toolTip">
          <string>Keep aspect ratio on scaled displays?</string>
         </property>
         <property name="text">
          <string>Keep aspect</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxShowFps">
         <property name="toolTip">
          <string>Show number of frames per second?</string>
         </property>
         <property name="text">
          <string>Show FPS</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelCompression">
         <property name="text">
          <string>Compression</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLCDNumber" name="lcdNumberCompression">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="numDigits">
          <number>1</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelQuality">
         <property name="text">
          <string>Quality</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLCDNumber" name="lcdNumberQuality">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="numDigits">
          <number>1</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelEncoding">
         <property name="text">
          <string>Encoding</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="3">
        <widget class="QComboBox" name="comboBoxEncoding">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="statusTip">
          <string>Select encoding</string>
         </property>
         <property name="whatsThis">
          <string>Select encoding</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelRenderer">
         <property name="text">
          <string>Renderer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <widget class="QComboBox" name="comboBoxRenderer">
         <property name="toolTip">
          <string>Select renderer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QSlider" name="sliderQuality">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Set quality level (0 - 9)</string>
         </property>
         <property name="statusTip">
          <string>Set quality level (0 - 9)</string>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QSlider" name="sliderCompression">
         <property name="toolTip">
          <string>Set compression level (0 - 9)</string>
         </property>
         <property name="statusTip">
          <string>Set compression level (0 - 9)</string>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBoxLanguage</tabstop>
  <tabstop>comboBoxStyle</tabstop>
  <tabstop>fontComboBoxAppFont</tabstop>
  <tabstop>spinBoxAppFontSize</tabstop>
  <tabstop>checkBoxMaximizeWindows</tabstop>
  <tabstop>checkBoxNativeFileDialogs</tabstop>
  <tabstop>checkBoxQuiet</tabstop>
  <tabstop>checkBoxScaledDisplay</tabstop>
  <tabstop>comboBoxRenderer</tabstop>
  <tabstop>checkBoxBilinearFiltering</tabstop>
  <tabstop>comboBoxEncoding</tabstop>
  <tabstop>checkBoxKeepAspect</tabstop>
  <tabstop>sliderQuality</tabstop>
  <tabstop>checkBoxShowFps</tabstop>
  <tabstop>sliderCompression</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberQuality</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCompression</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberCompression</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
